macros/unpack/get_column_values.sql

```diff
@@ -0,0 +1,39 @@
+{%- macro get_column_values(node_type) -%}
+    {{ return(adapter.dispatch('get_column_values', 'dbt_project_evaluator')(node_type)) }}
+{%- endmacro -%}
+
+{%- macro default__get_column_values(node_type) -%}
+
+    {%- if execute -%}
+        {%- if node_type == 'nodes' %}
+            {% set nodes_list = graph.nodes.values() %}   
+        {%- elif node_type == 'sources' -%}
+            {% set nodes_list = graph.sources.values() %}
+        {%- else -%}
+            {{ exceptions.raise_compiler_error("node_type needs to be either nodes or sources, got " ~ node_type) }}
+        {% endif -%}
+
+        {%- set values = [] -%}
+
+        {%- for node in nodes_list -%}
```

Comment:
per the question about joining, if we wanted to include some node level info, we also have the option of grabbing these values here from the `node` itself.

Reply:
@dave-connors-3 

Thanks for flagging. It was implemented like this in the past:
https://github.com/dbt-labs/dbt-project-evaluator/pull/386#discussion_r1373413986

But, as suggested by @b-per, we transitioned to the explicit join. I believe it makes more sense because we centralize the extraction of nodes' information in a single place. This ensures we maintain a single source of truth for the node's data and adhere to the DRY principle.